### PR TITLE
Update ChangeLog.md

### PR DESCRIPTION
**Pull request by kerautret**
*Thanks a lot for contributing to DGtalTools, before submitting your PR, please fill up the description and make sure that all checkboxes are checked.*

# PR Description

*your description here*

# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...).
- [ ] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ ] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -17,7 +16,7 @@
   - vol2slice: fix infinite slices extraction.
     (Bertrand Kerautret [#439](https://github.com/DGtal-team/DGtalTools/pull/439)) 
 
-- *generatos*
+- *generators*
   - 2dSimplePolygonDigitizer: new tool compute the Gauss Digitization of a simple closed polyline.
```
Comment (JacquesOlivierLachaud) [inline ChangeLog.md:20]: new tool *to* compute

Comment (kerautret) [reply ChangeLog.md:20]: done thanks!

**Comment by kerautret** [comment]
also corrected in the draft v1.3

**Comment by kerautret** [comment]
Can we merge ? just to make the release ;) @JacquesOlivierLachaud  or @dcoeurjo

**Review by dcoeurjo — APPROVED** [review]
All good

**Comment by kerautret** [comment]
Thanks ;)

**Pull request merged**